crates/sui-rpc-loadgen/src/payload/pay_sui.rs

```diff
@@ -21,16 +21,16 @@ impl<'a> ProcessPayload<'a, &'a PaySui> for RpcCommandProcessor {
         let clients = self.get_clients().await?;
         let SignerInfo {
             encoded_keypair,
-            signer_address,
             gas_budget,
             gas_payment,
         } = signer_info.clone().unwrap();
         let recipient = SuiAddress::random_for_testing_only();
-        let amount = 11;
+        let amount = 1;
```

Comment:
Does this mean send one coin? Was it arbitrarily 11 before?

Reply:
It means it will send 1 MIST. Yeah it's an arbitrary number